lib/ecto/adapters/clickhouse/structure.ex

```diff
@@ -3,10 +3,23 @@ defmodule Ecto.Adapters.ClickHouse.Structure do
   alias Ch.Query
   alias Ch.Connection, as: Conn
 
+  require Logger
```

Comment:
Logger doesn't seem to be used anymore.

Reply:
OK if I add some styling changes?

Reply:
I removed Logger and sure go for it! :)

Reply:
https://github.com/Zarathustra2/chto/pull/1